index.bs

```diff
@@ -1439,19 +1437,18 @@ object is discarded in the runtime, subsequent attempts to access it via the
 protocol will result in an error.
 
 <div algorithm>
-To get the <dfn>object id for an object</dfn> given a |session| and |object|:
+To get the <dfn>object id for an object</dfn> given an |object| in a given |realm|:
```

Comment:
similar comment here, I'd say `given |realm| and |object|`

Reply:
done